### DISCUSSION TITLE
Reorder CP nav sections

### DISCUSSION THREAD

**Discussion by MtDalPizzol** [statamic/cms#7399] [Q&A]
Hi folks. Following the docs and some GH investigation, I was able to create the navigation items I need. However, I couldn't find how to sort the sections in the nav. I'd like my custom section to be shown after the "Content" section, but it is rendered after the "Users" one.

Is it possible?

![image](https://user-images.githubusercontent.com/3843774/214131869-1212150b-1db4-4810-8e1c-6c9c3a938cfb.png)

**Comment by duncanmcclean** [comment]
It's coming in Statamic 3.4 😄

**Reply by MtDalPizzol** [reply]
Great. I'll leave my code on hold for now. Ain't going into production before 3.4, probably. 😅
